src/os/versionControl/components/VersionControlEditor.tsx

```diff
@@ -334,6 +369,25 @@ export const VersionControlEditor: React.FC<{
 
   const branches = doc.branchMetadata.branches ?? [];
 
+  // Currently we can't filter out comments that didn't exist in a previous version of the document
```

Comment:
Curious if there's a plan to update the Cursors API in Automerge soon to better handle deleted cursors? cc @alexjg